scenarios/kubernetes_janitor.py

```diff
@@ -105,7 +105,12 @@ def check_ci_jobs():
 
     match_re = re.compile(r'--gcp-project=(.+)')
     for value in config.values():
+        clean_hours = 24
         for arg in value.get('args', []):
+            # lifetime for soak cluster should be 7 days
```

Comment:
This is from the old legacy job, which gets redeployed every 7 days?

Reply:
that's from https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L156, I was trying to keep the same behavior